### ISSUE TITLE
[Bug]  bbox internvl

### ISSUE THREAD

**Issue by lixianyan2023** [OpenGVLab/InternVL#941]
### Checklist

- [ ] 1. I have searched related issues but cannot get the expected help.
- [ ] 2. The bug has not been fixed in the latest version.
- [ ] 3. Please note that if the bug-related issue you submitted lacks corresponding environment info and a minimal reproducible demo, it will be challenging for us to reproduce and resolve the issue, reducing the likelihood of receiving feedback.

### Describe the bug

我得出了目标信息的bbox，我要怎么画出他的框呢？如何获取原始图像的高和宽

### Reproduction

我得出了目标信息的bbox，我要怎么画出他的框呢？如何获取原始图像的高和宽

### Environment

```Shell
no
```

### Error traceback

```Shell

```

**Comment by lixianyan2023** [comment]
def collate_fn(batches):
    pixel_values = torch.cat([_['pixel_values'] for _ in batches], dim=0)
    texts = [_['text'] for _ in batches]
    bboxes = [_['bbox'] for _ in batches]
    hws = [_['hw'] for _ in batches]
    return pixel_values, texts, bboxes, hws
  这里面的hws是image.size得到的吗?

**Comment by yuecao0119** [comment]
你好，

你可以通过PIL库打开原始图片，然后获得宽高
```python
from PIL import Image

# 打开图像
image = Image.open('your_image.jpg')

# 获取图像的宽度和高度
width, height = image.size
```
